orga/tests/acceptance/student-list-test.js

```diff
@@ -130,31 +130,31 @@ module('Acceptance | Student List', function(hooks) {
 
       module('when student authenticated by username and email', async function() {
 
-        test('it should display password reset modal button for student', async function(assert) {
+        test('it should have a button to display actions on the student', async function(assert) {
           // when
           await visit('/eleves');
 
           // then
-          assert.dom('.table tbody tr:nth-child(6) td:last-child button .fa-cog').exists();
+          assert.dom('[aria-label="Afficher les actions pour cet élève"]').exists();
         });
 
         test('it should open modal and display password reset button', async function(assert) {
           // given
           await visit('/eleves');
 
           // when
-          await click('.table tbody tr:nth-child(6) td:last-child button');
+          await click('[aria-label="Afficher les actions"]');
+          await click('li');
 
           // then
-          assert.dom('.pix-modal-overlay').exists();
-          assert.dom('.pix-modal-footer button').exists();
-          assert.dom('.pix-modal-footer button').hasText('Réinitialiser le mot de passe');
+          assert.contains('Réinitialiser le mot de passe');
         });
 
         test('it should display unique password input when reset button is clicked', async function(assert) {
           // given
           await visit('/eleves');
-          await click('.table tbody tr:nth-child(6) td:last-child button');
+          await click('[aria-label="Afficher les actions"]');
+          await click('li');
```

Comment:
[click](https://github.com/emberjs/ember-test-helpers/blob/master/API.md#parameters-1) peut prendre un élément en premier paramètre. Je pense qu'on peut donc utiliser `contains` (il faudrait pouvoir l'importer pour ces cas là) pour s'abstraire de l'implémentation :
```suggestion
          await click(contains('Gérer le compte'));
```

Qu'en pensez-vous ?

Reply:
En fait, je ne crois pas que c'est possible, car l'implémentation dans QUnit a été réalisée sur `assert` de QUnit et retourne juste le résultat de l'assertion (OK ou KO). Ce n'est pas un sélecteur, et il  ne retourne pas l'élément.

Pour info,  voilà l'implémentation du contains pour QUnit :
```js
export function contains(text) {
  const elements = getChildrenThatContainsText(getRootElement(), text);
  const result = elements.length > 0;

  let message = `There is no elements with "${ text }"`;
  if (result) {
    message = `Element with "${ text }" found`;
  }

  this.pushResult({
    result,
    message
  });
}
```

Reply:
On pourrait avoir une version vraiment exportable de la fonction qui fait ?

```js
const elements = getChildrenThatContainsText(getRootElement(), text);
if (elements.length) return elements[0]
return null
```

Reply:
J'ai essayé de faire quelque chose dans le genre, mais ça va demander plus de travail, car la fonction `getChildrenThatContainsText` ne renvoie pas forcement l'élément le plus bas (dans notre cas le `li` qui est cliquable), mais il renvoie le premier élément qui contient le texte parmi ces descendants (dans notre cas le `td`)